modules/ROOT/pages/endpoint-as2-receive.adoc

```diff
@@ -1,75 +1,81 @@
 = AS2 Receive Endpoint Settings
 
-AS2 endpoint settings specify where to receive inbound files from your trading partners. You can use the same AS2 receive endpoints to receive inbound transmissions from multiple partners.
+AS2 endpoint settings dictate where to receive inbound files from your trading partners. You can use the same AS2 receive endpoints to receive inbound transmissions from multiple partners.
 
-Create the AS2 identifiers for your organization and your trading partners in the respective host and trading partner profile pages.
+You can create the AS2 identifiers for your organization and for your trading partners in the respective xref:configure-host.adoc[host] and xref:configure-partner.adoc[trading partner] profile pages.
 
 [%header%autowidth.spread]
 |===
-|Setting |Description | Required
-|Endpoint Name
-|Name used to easily identify the purpose of this endpoint. The name is auto-populated, but can be overwritten. 
-|Yes 
+| Setting | Description | Required
+| Endpoint Name
+| The name used to identify the endpoint. The `Endpoint Name` is automatically generated, but can be modified manually.
+| Yes
 
-|Description
-|Describe any important details for this endpoint. 
+| Owner
+| The trading partner or host that owns the endpoint. 
+| Yes
+
+| Protocol
+| The protocol selected in the `Protocol` field of the *New endpoint: Receive from partners* window. The value in this field cannot be modified.
+| Yes
+
+| Description
+| The user-defined field for describing the purpose and use of the endpoint.
 | No
 
 | AS2 URL
 a| The HTTP or HTTPS address where you receive AS2 messages.
```

Comment:
"you" being the host organization?
GLOBALLY standardize/clarify. I will stop marking it, so please search for and doublecheck wording?

Reply:
The following text is an inordinate amount for inclusion in a table of values that have heretofore been brief and scannable, as they should be. I suggest seeking a solution that moves this outside of the table columns, perhaps following the table as further explanation of how the URL works in this scenario?

Reply:
Reviewed for "you"/"your". Thanks!

Moved URL section in all topics.